lib/v2/youzhiyouxing/materials.js

```diff
@@ -15,15 +15,15 @@ module.exports = async (ctx) => {
 
     const $ = cheerio.load(response.data);
 
-    let items = $('li[phx-click="material"]')
+    let items = $('li[id*="material"]')
         .toArray()
         .map((item) => {
             item = $(item);
 
             return {
                 title: item.text(),
-                link: `${rootUrl}/materials/${item.attr('phx-value-id')}`,
-                pubDate: parseDate(item.find('.tw-text-right').text(), ['YYYY年M月D日', 'M月D日']),
+                link: `${rootUrl}/${item.find('a').attr('href')}`,
```

Comment:
```suggestion
                link: `${rootUrl}${item.find('a').attr('href')}`,
```
https://github.com/DIYgod/RSSHub/runs/7374250344?check_suite_focus=true#step:11:34